editor.planx.uk/src/@planx/components/Checklist/Editor/components/GroupedOptions.tsx

```diff
@@ -51,7 +52,7 @@ export const GroupedOptions = ({
             <Box display="flex" pb={1}>
               <InputRow>
                 <Input
-                  errorMessage={`formik.errors[${groupIndex}].title`}
```

Comment:
Problems with this original implementation - 
 - It's a string not a variable!
 - It's the wrong path